docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Install the microservice Helm chart 
+menuTitle: Install microservice Loki
+description: Installing Loki in microservice (distributed) mode using the Helm chart.
+aliases:
+  - ../../../installation/helm/microservice/
+  - ../../../installation/helm/install-microservice/
+weight: 300
+keywords: 
+---
+
+# Install the microservice Helm chart
+<!-- vale Grafana.Quotes = NO -->
+<!-- vale Grafana.Quotes = YES -->
```

Comment:
These are only necessary if you're skipping a Vale rule.  Since there's nothing between the comments, you're not skipping a rule and can remove these.